package.json

```diff
@@ -115,11 +115,14 @@
     "@types/chai": "^4.2.14",
     "@types/glob": "^7.1.3",
     "@types/mocha": "^8.2.0",
+    "@types/mock-fs": "^4.13.0",
+    "@types/ncp": "^2.0.4",
     "@types/node": "^10.0.0",
     "@types/semver": "^7.3.4",
     "@types/sinon": "^9.0.10",
     "@types/tmp": "^0.2.0",
     "@types/vscode": "^1.29.0",
+    "@types/yauzl": "^2.9.1",
```

Comment:
you installed the `@types/yazul` but where is the library itself? Why isn't it a dependency as well?

Reply:
for some reason this is required in order to use `extract-zip` when compiled on linux